internal/shim-sgx/src/uarch.rs

```diff
@@ -70,17 +68,17 @@ impl Default for REPORT_PAYLOAD {
 /// signed with the report key. Used by the target enclave to verify the source
 /// enclave by grabbing the report key with EGETKEY and calculating CMAC.
 #[repr(C, align(512))]
-pub struct REPORT {
-    pub body: REPORT_PAYLOAD,
+pub struct Report {
+    pub payload: ReportPayload,
     pub keyid: [u8; 32],
     pub mac: [u64; 2],
     padding: [u8; 80],
 }
 
-impl Default for REPORT {
+impl Default for Report {
     fn default() -> Self {
-        REPORT {
-            body: REPORT_PAYLOAD::default(),
+        Report {
```

Comment:
nit; this could also be
```suggestion
        Self {
```

Reply:
I'll fix it.